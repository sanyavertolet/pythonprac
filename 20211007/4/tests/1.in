'sin(x^2)', 'log(x^5)', 'x**5/y**(1/6)'
228
