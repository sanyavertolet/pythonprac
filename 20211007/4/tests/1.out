-0.0669651364494781
